lib/std/meta.zig

```diff
@@ -1200,6 +1200,8 @@ pub inline fn hasUniqueRepresentation(comptime T: type) bool {
         .Array => |info| hasUniqueRepresentation(info.child),
 
         .Struct => |info| {
+            if (info.layout == .@"packed") return @sizeOf(T) * 8 == @bitSizeOf(T);
```

Comment:
Would it be better to use `byte_size_in_bits` over `8` here based on this comment:

https://github.com/ziglang/zig/blob/d74180c373274bc545992c2a96a8e7618f8bd52c/lib/std/mem.zig#L23-L29

Apologies if I'm misunderstanding anything :)